README.md

```diff
@@ -49,13 +49,15 @@ The version command takes no arguments.
 #### Signing
 
 ```
-cfssl sign [-ca cert] [-ca-key key] hostname csr [subject]
+cfssl sign [-ca cert] [-ca-key key] [-hostname hostname] csr [subject]
```

Comment:
If sign also supports comma-separated hostnames, add it here. Otherwise we should include support by re-using the code from gencert.